src/backend/distributed/citus--7.4-2--7.4-3.sql

```diff
@@ -10,4 +10,88 @@ COMMENT ON FUNCTION worker_hash_partition_table(bigint, integer, text, text, oid
                                                 anyarray)
     IS 'hash partition query results';
 
+CREATE FUNCTION citus_connections_hash()
```

Comment:
Could we put these in the citus schema and make sure regular users cannot call them?